### DISCUSSION TITLE
Is there a way to get the semantic classes of segmented regions?

### DISCUSSION THREAD

**Discussion by ogencoglu** [opengeos/segment-geospatial#355] [Q&A] [answered]
It seems like this is not a fine-tuned version of SAM with class annotations. It just outputs semantic regions and can not assign class tags to it such as tree, building, road etc.

Is there a way to use only the vision model (no text prompting or anything) to segment the image and also get the class annotations?

**Comment by giswqs** [comment, marked as answer]
No, it does not exist yet. You might want to use some other models dedicated for land cover classification.

**Reply by ogencoglu** [reply]
Thanks for the swift reply! Any pointers for such pretrained models (if you happen to know by memory quickly)?
